package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -232,7 +238,9 @@ def per_core_work(start_frame_coord_array, end_frame_coord_array, dictionary_cub
     list_previous_frame_indices = []
     # define some utility functions for trajectory iteration:
 
-    def point_in_cube(array_point_coordinates, list_cube_vertices, cube_centroid):
+    # typing: numpy
+    def point_in_cube(array_point_coordinates: Sequence[float], list_cube_vertices: Sequence[float],
+                      cube_centroid) -> np.ndarray:
```

Comment:
```suggestion
    def point_in_cube(
        array_point_coordinates: Sequence[float],
        list_cube_vertices: Sequence[float],
        cube_centroid: np.ndarray,
    ) -> np.ndarray:
```

`cube_centroid` was not annotated, from what I see it is an array.